app/src/components/ArtworkInfo.tsx

```diff
@@ -31,6 +31,8 @@ class ArtworkInfo extends React.Component<ArtworkInfoProps, {}> {
                 className={this.props.classes.inline}
                 color="textPrimary"
               >
+                {/* image will be at https://ipfs.io/ipfs/METAURI */}
+                {this.props.artwork.metaUri} +
```

Comment:
Think you need the https://ipfs.io/ipfs/ to be in the string according to the ERC721 metadata information